spotlight-client/src/contentApi/sources/us_pa.ts

```diff
@@ -115,6 +116,7 @@ const content: TenantContent = {
     },
     Parole: {
       title: "Parole",
+      previewTitle: "Returns from Parole, by Age Group",
```

Comment:
similarly, we are not showing the age breakdowns in this preview, so it should say "by Reason" instead